crates/bevy_tasks/Cargo.toml

```diff
@@ -72,7 +72,7 @@ crossbeam-queue = { version = "0.3", default-features = false, features = [
 [target.'cfg(target_arch = "wasm32")'.dependencies]
 wasm-bindgen-futures = { version = "0.4", optional = true }
 pin-project = { version = "1", optional = true }
-futures-channel = { version = "0.3", optional = true }
+async-channel = { version = "2.5.0", optional = true }
```

Comment:
2.3.0 is used elsewhere in tree

Reply:
Actually, this is *already* an option adept on line 64. I don't think this line is needed at all.

Reply:
Woops! Nice that we can converge on this impl for more things.

Reply:
Quick, add a "removes dependency" label to the PR